### PR TITLE
adds the package type "yawik-module"

### PR DESCRIPTION
**Pull request by cbleek**
Hi,

I would like to add the package type "yawik-module"

regards,

Carsten

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "composer/installers",
+    "name": "cross-solution/installers",
```
Comment (niksamokhvalov) [inline composer.json]: What are you doing? :)

Comment (cbleek) [inline composer.json]: Am 15.04.16 um 22:22 schrieb Nik Samokhvalov:
>
> In composer.json
> <https://github.com/composer/installers/pull/296#discussion_r59935448>:
>
> > @@ -1,5 +1,5 @@
> >  {
> > -    "name": "composer/installers",
> > +    "name": "cross-solution/installers",
>
> What are you doing? :)
>
oh God. I've changed the name of the package to test against my fork.
Did not know that this affects my pull request. I've fixed that in my
fork now.

Thank you for the review.

**Comment by niksamokhvalov** [comment]
Add your installer to the [`InstallerTest::dataForTestSupport`](https://github.com/composer/installers/blob/master/tests/Composer/Installers/Test/InstallerTest.php#L84), please.

**Comment by niksamokhvalov** [comment]
@cbleek fix conflicts, please.

**Comment by cbleek** [comment]
I think, the problem was my commit https://github.com/composer/installers/pull/296/commits/173765f848f3c9ab2b273c96b9cfc83b32161b85

I'll create a new pull request

**Pull request closed without merging**